test/groovy/org/ods/util/ProjectSpec.groovy

```diff
@@ -655,41 +655,103 @@ class ProjectSpec extends SpecHelper {
         def result = project.loadMetadata()
 
         then:
-        def expected = [
-            id: "myId",
-            name: "myName",
-            description: "myDescription",
-            repositories: [
-                [
-                    id: "A",
-                    url: "https://github.com/my-org/my-repo-A.git",
-                    branch: "master",
-                    type: MROPipelineUtil.PipelineConfig.REPO_TYPE_ODS_CODE,
-                    data: [
-                        documents: [:]
-                    ]
-                ],
-                [
-                    id: "B",
-                    url: "https://github.com/my-org/my-repo-B.git",
-                    branch: "master",
-                    type: MROPipelineUtil.PipelineConfig.REPO_TYPE_ODS_CODE,
-                    data: [
-                        documents: [:]
-                    ]
-                ],
-                [
-                    id: "C",
-                    url: "https://github.com/my-org/myid-C.git",
-                    branch: "master",
-                    type: MROPipelineUtil.PipelineConfig.REPO_TYPE_ODS_CODE,
-                    data: [
-                        documents: [:]
-                    ]
-                ]
-            ],
-            capabilities: []
-        ]
+        def expected = new JsonSlurperClassic().parseText("""
```

Comment:
@georgfedermann why JSON? This suggests that we somewhere use a document using this format that has some relevance (contract), but this is not the case. Here, we test the final result after parsing a metadata file, plus any default values added by the reader implementation. Can you please revert to the original data structure layout?